tests/qchem/of_tests/test_molecular_hamiltonian.py

```diff
@@ -153,11 +153,12 @@ def test_building_hamiltonian_molecule_class(
 
 
 @pytest.mark.parametrize(
-    ("symbols", "geometry", "h_ref_data"),
+    ("symbols", "geometry", "mapping", "h_ref_data"),
```

Comment:
We might need to add "method" here to make sure we check both "dhf" and "pyscf" backends for different mappings.